app/models/foreman_salt/concerns/host_managed_extensions.rb

```diff
@@ -140,13 +163,11 @@ def generate_provisioning_key
         SecureRandom.hex(10)
       end
 
-      def create_salt_autosign
-        Rails.logger.info("Create salt autosign key for host #{fqdn}")
+      def configure_salt_autosign
+        Rails.logger.info("Configure salt autosign key for host #{fqdn} on #{salt_proxy.url}")
```

Comment:
What happens if no salt_proxy is configured?

Reply:
The code shall only be executed when a `salt_proxy` is configured: https://github.com/theforeman/foreman_salt/pull/211/files#diff-179b69b42b6edeaca2c6102e7522527493004e93a18e24f539fb91d6cfdc04cdR45